rust/stackable-cockpit/src/platform/operator/mod.rs

```diff
@@ -213,7 +213,7 @@ impl OperatorSpec {
         let mut helm_values = None;
         if self.name == "listener" {
             helm_values = Some(
-                LISTENER_OPERATOR_PRESET.get()
+                LISTENER_CLASS_PRESET.get()
                     .expect("At this point LISTENER_OPERATOR_PRESET must be set by determine_and_store_listener_operator_preset")
```

Comment:
Sorry, I missed this one earlier.

```suggestion
                    .expect("At this point LISTENER_CLASS_PRESET must be set by determine_and_store_listener_class_preset")
```